sdk/identity/Azure.Identity/src/IdentityCompatSwitches.cs

```diff
@@ -7,6 +7,8 @@ namespace Azure.Identity
 {
     internal class IdentityCompatSwitches
     {
+        internal const string AllowMultiTenantAuthExecutionEnvVar = "AZURE_IDENTITY_ALLOW_MULTITENANT_AUTH";
+        internal const string AllowMultiTenantAuthSwitchName = "Azure.Identity.AllowMultiTenantAuth";
```

Comment:
Should this be called 'DisableMultiTenantAuth` instead since it disables the feature updates?